src/index.ts

```diff
@@ -6,35 +6,41 @@ import {Zcl} from 'zigbee-herdsman';
 
 import fromZigbee from './converters/fromZigbee';
 import toZigbee from './converters/toZigbee';
-import allDefinitions from './devices';
 import * as configureKey from './lib/configureKey';
 import * as exposesLib from './lib/exposes';
 import {Enum as EnumClass} from './lib/exposes';
 import {generateDefinition} from './lib/generateDefinition';
-import * as logger from './lib/logger';
+import {logger} from './lib/logger';
 import {
     Configure,
     Definition,
     DefinitionExposes,
     DefinitionExposesFunction,
-    DefinitionWithExtend,
     Expose,
     Fingerprint,
+    IndexedDefinition,
     KeyValue,
     OnEvent,
     OnEventData,
     OnEventMeta,
     OnEventType,
     Option,
+    ResolvedDefinition,
     Tz,
     Zh,
 } from './lib/types';
 import * as utils from './lib/utils';
+import modelsIndexJson from './models-index.json';
```

Comment:
huh, neat trick I didn't know about, I always JSON.parse(fs.readFileSync(...

Reply:
Thanks to this: https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/tsconfig.json#L16 😉

https://www.typescriptlang.org/tsconfig/#resolveJsonModule